commerce-cloud-code/app_yourshophere/cartridge/partials/account/welcomeBack.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Creates a model for the welcome back partial
+ * @param {Object} params - The parameters object containing customer and settings
+ * @param {dw.customer.Customer} params.customer - The authenticated customer object
+ * @param {Object} [params.settings] - Optional settings from the component
+ * @returns {Object} The model for the welcome back message
+ */
+const createModel = (params) => {
+    const Resource = require('dw/web/Resource');
+    const URLUtils = require('dw/web/URLUtils');
+    const customer = params && params.customer;
+    let firstName = '';
+    if (customer && customer.profile && customer.profile.firstName) {
+        firstName = customer.profile.firstName;
+    }
+    return {
+        firstName,
+        isAuthenticated: customer && customer.authenticated,
+        message: firstName
```

Comment:
The label could be a page designer config, right? Not sure how one could nicely handle placeholders to change the order in a sentence.

Reply:
Yes, good idea.

Reply:
Done